root/components/relationship-editor.tt

```diff
@@ -116,6 +116,13 @@
       <!-- /ko -->
     </td>
   </tr>
+  <!-- ko if: isUntypedSeries() -->
+    <tr>
+      <td colspan="2">
+        [% l('To add parts to this series, please select a type first') %]
```

Comment:
Maybe specify series type here in case it's confused with relationship type, since that's in the currently open dialog?
```suggestion
        [% l('To add parts to this series, please select a series type first.') %]
```